x-pack/plugins/security_solution/kibana.jsonc

```diff
@@ -53,6 +53,7 @@
       "notifications",
       "savedSearch",
       "unifiedDocViewer",
+      "charts"
```

Comment:
The side bar fields stats use a charts module, so `charts` have become a required plugin now.